src/utils/helper-functions.tsx

```diff
@@ -180,7 +180,7 @@ export const spliceSx = (
   ({
     ...defaultStyles,
     ...newStyles
-  } as SxProps)
+  }) as SxProps
```

Comment:
fix this error, looks like you have some troubles with local prettier and lint

Reply:
fixed